Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```
@@ -183,8 +180,8 @@ div.bocList table.bocListTable td.bocListDataCell *.bocListCellStructureElement
 div.bocList table.bocListTable th.bocListTitleCell > a,
 div.bocList table.bocListTable td.bocListDataCell > a,
 div.bocList table.bocListTable td.bocListDataCell *.bocListCellStructureElement > a,
-div.bocList table.bocListTable td.bocListDataCell > span:not(.bocListCellStructureElement),
-div.bocList table.bocListTable td.bocListDataCell > span:not(.bocListCellStructureElement),
+div.bocList table.bocListTable td.bocListDataCell > div:not(.bocListCellStructureElement),
+div.bocList table.bocListTable td.bocListDataCell > div:not(.bocListCellStructureElement),
```

```suggestion
div.bocList table.bocListTable td.bocListDataCell > div:not(.bocListCellStructureElement),
```
I think the original code has a copy&paste duplicate here.